src/test/java/io/odpf/firehose/sink/mongodb/util/MongoSinkClientUtilTest.java

```diff
@@ -0,0 +1,115 @@
+package io.odpf.firehose.sink.mongodb.util;
+
+import com.mongodb.ServerAddress;
+import io.odpf.firehose.metrics.Instrumentation;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class MongoSinkClientUtilTest {
```

Comment:
should move the test to factory once we move the functionality there.

Reply:
fixed